src/F0_output.py

```diff
@@ -134,44 +175,46 @@ def store_timeseries_all_busses_to_excel(dict_values):
     return
 
 
-def store_as_json(dict_values, file_name):
+def convert(o):
     """
-
-    :param dict_values:
-    :param file_name:
-    :return:
+    This converts all data stored in dict_values that is not compatible with the json format to a format that is compatible.
+    :param o: Any type. Object to be converted to json-storable value.
+    :return: json-storable value.
     """
+    if isinstance(o, numpy.int64):
+        return int(o)
+    # todo this actually drops the date time index, which could be interesting
+    if isinstance(o, pd.DatetimeIndex):
+        return "date_range"
+    # if isinstance(o, pd.datetime):
+    #    return str(o)
+    if isinstance(o, pd.Timestamp):
+        return str(o)
+    # todo this also drops the timeindex, which is unfortunate.
+    if isinstance(o, pd.Series):
+        return "pandas timeseries"  # o.values
+    if isinstance(o, numpy.ndarray):
+        return "numpy timeseries"  # o.tolist()
+    if isinstance(o, pd.DataFrame):
+        return "pandas dataframe"  # o.to_json(orient='records')
+    logging.error(
```

Comment:
You don't want to raise an actual error? That way we can also test the error was correctly raised?

Reply:
But doesnt this raise an error in ll. 206? I might edit this anyway.